versioned/tiered/mongodb/pom.xml

```diff
@@ -35,12 +35,6 @@
       <artifactId>nessie-versioned-tiered-impl</artifactId>
       <version>${project.version}</version>
     </dependency>
-    <dependency>
```

Comment:
Why changes to Mongo?

Reply:
This looks like your change on 2021-01-18  4e5b3b7f7248fac79219e3dd6b2005ad24620f45